tests/unit/src/commands/initialize/initAuthorization.ts

```diff
@@ -0,0 +1,184 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import loadModule, { cleanupModuleMocks } from '../../../../_support/loadModule';
+import { spy, stub, SinonSpy, SinonStub } from 'sinon';
+
+let initAuthorization: any;
+let Travis: any;
+let GitHub: any;
+let TravisSpy: SinonSpy;
+let GitHubSpy: SinonSpy;
+let isAuthorizedStub: SinonStub;
+let travisCreateAuthorizationStub: SinonStub;
+let travisDeleteAuthorizationStub: SinonStub;
+let listEnvironmentVariablesStub: SinonStub;
+let setEnvironmentVariablesStub: SinonStub;
+let fetchRepositoryStub: SinonStub;
+let authenticateStub: SinonStub;
+let getRateLimitStub: SinonStub;
+let toStringStub: SinonStub;
+let repoCreateAuthorizationStub: SinonStub;
+let repoDeleteAuthorizationStub: SinonStub;
+let findStub: SinonStub;
+let githubAuthStub: SinonStub;
+
+registerSuite({
+	name: 'commands/initialize/initAuthorization',
+
+	before() {
+		isAuthorizedStub = stub();
```

Comment:
Addressed in d0befb7. Good thing you asked me to address this, as there was another minor bug in the `afterEach` where I reassigned two stubs rather than call `reset` on them!